Projects/Claims/Sources/Models/FlowClamSingleItemStepModel.swift

```diff
@@ -34,6 +35,16 @@ public struct FlowClamSingleItemStepModel: FlowClaimStepModel {
         self.currencyCode = data.purchasePrice?.currencyCode.rawValue
         self.selectedItemBrand = data.selectedItemBrand
         self.selectedItemModel = data.selectedItemModel
+
+        if self.selectedItemModel == nil && self.customName == nil {
+            let currentDeviceName = "Apple " + UIDevice.modelName
+            if let matchingModelWithCurrentDevice = self.availableItemModelOptions.first(where: {
+                $0.displayName == currentDeviceName
+            }) {
+                self.selectedItemModel = matchingModelWithCurrentDevice.itemModelId
+            }
+        }
+
```

Comment:
Maybe better
```
            let currentDeviceName = UIDevice.modelName.lowercased()
            if let matchingModelWithCurrentDevice = self.availableItemModelOptions.first(where: {
                let name = $0.displayName.lowercased().replacingOccurrences(of: "apple", with: "").trimmingCharacters(in: .whitespacesAndNewlines)
                return name == currentDeviceName
            }) {
                self.selectedItemModel = matchingModelWithCurrentDevice.itemModelId
                self.selectedItemBrand = matchingModelWithCurrentDevice.itemBrandId
            }
```

Reply:
Why? When they remove apple we don't have to care since it will still work.

Reply:
we don't have apple in the string anymore so we don't need .replacingOccurrences(of: "apple", with: "")